recipes/django-rest-swagger/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "django-rest-swagger" %}
+{% set version = "2.2.0" %}
+
+package:
+  name: django-rest-swagger
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/d/django-rest-swagger/django-rest-swagger-{{ version }}.tar.gz
+  sha256: 48f6aded9937e90ae7cbe9e6c932b9744b8af80cc4e010088b3278c700e0685b
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python-coreapi
+    - djangorestframework >=3.5.4
+    - openapi-codec >=1.3.1
```

Comment:
@lsetiawan looks like only this one is missing, right?

Reply:
[`openapi-codec`](https://anaconda.org/search?q=openapi-codec) is too old and it's only for osx. I guess I need to create recipe for that. Alright. Thanks for helping me! :smile: